src/OnePlusBot/Modules/showemoji.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Discord;
```

Comment:
I understand compatibility for mobile devices, however, this might be abused.  Testing of usage shall determine its verdict.